react/what-is-props/what-is-props.md

```diff
@@ -0,0 +1,131 @@
+# What is props?
+
+---
+
+If you are beginner in react you must have heard about `props` a lot, so let's understand, how is it so important to understand.
+
+## How is it useful?
+
+Before starting we need to understand how react works. So React is a component based `JavaScript` Library where we can divide our project into various reusable components.
+
+For example the below code is your main `App.js` file which you can see when we create a project using create-react-app
+
+```JavaScript
+import "./styles.css";
+import Greet from "./Greet";
+
+function App() {
+  var Humans = [
+    {
+      id: 1,
+      name: "Ashray",
+      age: 22
+    },
+    {
+      id: 2,
+      name: "Aman",
+      age: 25
+    }
+  ];
+
+  return (
+    <div className="App">
+      <Greet Humans={Humans} />
+    </div>
+  );
+}
+export default App;
+```
+
+In the above code we can see that I have created one `Component`.
+
+- Greet
+
+There is an array of object's with key value pair of `id`, `name` and `age`
+
+1. Now let's come to our main topic **Props** it stands for **properties** it's not mandatory to write only `props` we can name it anything we wish for.
```

Comment:
This point should come late. First explain what a prop is, then what we also use for naming. I'm sure you can do it with some more research on the topic.